gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -139,6 +162,12 @@
         "board": BOARD,
         "serial_number": SERIAL,
         "config_main_stack_size": 4096,
+        "cmsis_path": "/content/cmsis"
```

Comment:
add parens around the ternary, here and below

Reply:
Replaced this to use `os.getenv` to be cleaner.